django-stubs/utils/translation/__init__.pyi

```diff
@@ -3,6 +3,7 @@ from contextlib import ContextDecorator
 from typing import Any, Callable, Optional, Union
 
 from django.core.handlers.wsgi import WSGIRequest
+from django.utils.functional import Promise
```

Comment:
I think that `Promise` is a new thing, isn't it? Looks like that latest `3.2.x` does not have this primitive: https://github.com/django/django/tree/3.2.6/django/utils

Reply:
Moreover, even in latest `master` it is just a plain class with no methods or anything: https://github.com/django/django/blob/main/django/utils/functional.py#L68

Reply:
Hmm.. `Promise` has been there for quite a while: https://github.com/django/django/commit/e4e28d907aee9f0e3977e2c6237406d7746c5719. But you are right that `Promise` is just a plain class. Since the proxy method knows about the `resultclasses`, I guess creating generics like `Promise[str]` would be better?

Reply:
Maybe we can add these methods to `Promise` as well? https://github.com/django/django/blob/3.2.6/django/utils/functional.py#L84-L191